### DISCUSSION TITLE
[Docs]: add an example on how to use poe_mode and poe_type parameters in netbox_device_interface module

### DISCUSSION THREAD

**Discussion by r000m5** [netbox-community/ansible_modules#884] [General]
### Change Type

Addition

### Area

Functionality/features

### Proposed Changes

Add an example on how to use poe_mode and poe_type parameters in netbox_device_interface module.
Indeed I can't succeed in using it, I always get an ansible error like `400 Bad Request: {'poe_mode': ['PD is not a valid choice.']}`

And I can't understand how to use it, since for the example provided 'PD' is a valid choice for poe_mode in the netbox GUI.

PS : if someone could provide an example in this issue's comments on how to use poe_mode and poe_type I would really appreciate it since I'm stuck with this for now.

**Comment by rodvand** [comment]
If you go to your NetBox instance and visit /api/docs/ you can find this under /dcim/interfaces/ (POST):
<img width="1232" alt="Screenshot 2022-11-27 at 12 30 43" src="https://user-images.githubusercontent.com/56864/204132952-7504f620-5346-4bb2-a6e4-c2b613e71807.png">

The choices in the Ansible modules does not necessarily correspond with the choices in the NetBox GUI.

**Reply by r000m5** [reply]
Thanks for the quick reply, very much appreciated especially since today is sunday.
I'll know where to look for the information next time !
